consul_exporter.go

```diff
@@ -86,6 +86,15 @@ func NewExporter(uri string) *Exporter {
 			[]string{"service", "node"},
 		),
 
+		nodeChecks: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "agent_check",
+				Help:      "Is this check passing on this node?",
```

Comment:
Yeah. I'm :+1: on just passing / not passing. Keeping just 0 and 1 also makes it much easier to `sum` and `count` to figure out how many nodes / checks are passing, failing, etc.